docs/user-guide/src/capm3/pivoting.md

```diff
@@ -0,0 +1,166 @@
+# CAPM3 Pivoting
+
+## What is pivoting
+
+Cluster-api-provider-metal3 (CAPM3) implements support for CAPI's 'move/pivoting' feature.
+
+CAPI Pivoting feature is a process of moving the provider components and declared Cluster API resources from a source
+management cluster to a target management cluster by using the `clusterctl` functionality called "move".
+More information about the general CAPI "move" functionality can be find [here](https://cluster-api.sigs.k8s.io/clusterctl/commands/move.html).
+
+In Metal3, pivoting is performed by using the CAPI `clusterctl` tool provided by Cluster-API. `clusterctl` recognizes pivoting as move.
+During the pivot process `clusterctl` pauses any reconciliation of Cluster-API objects and this gets propagated to (CAPM3) objects as well.
+Once all the objects are paused, the objects are created on the other side on the target cluster and deleted from the
+source cluster.
```

Comment:
Is the deletion always true ? and do we do it that way ?
@kashifest

Reply:
Yes